src/main/java/im/toduck/domain/diary/domain/service/DiaryService.java

```diff
@@ -0,0 +1,48 @@
+package im.toduck.domain.diary.domain.service;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import im.toduck.domain.diary.common.mapper.DiaryImageFileMapper;
+import im.toduck.domain.diary.common.mapper.DiaryMapper;
+import im.toduck.domain.diary.persistence.entity.Diary;
+import im.toduck.domain.diary.persistence.entity.DiaryImage;
+import im.toduck.domain.diary.persistence.repository.DiaryImageRepository;
+import im.toduck.domain.diary.persistence.repository.DiaryRepository;
+import im.toduck.domain.diary.presentation.dto.request.DiaryCreateRequest;
+import im.toduck.domain.user.persistence.entity.User;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DiaryService {
+	private final DiaryRepository diaryRepository;
+	private final DiaryImageRepository diaryImageRepository;
+
+	@Transactional
+	public Diary createDiary(
+		final User user,
+		final DiaryCreateRequest request
+	) {
+		Diary diary = DiaryMapper.toDiary(
+			user,
+			request.date(),
+			request.emotion(),
+			request.title(),
+			request.memo()
+		);
+		return diaryRepository.save(diary);
+	}
```

Comment:
p4: Mapper 메서드 를 개선해서
```suggestion
	@Transactional
	public Diary createDiary(
		final User user,
		final DiaryCreateRequest request
	) {
		Diary diary = DiaryMapper.toDiary(user, request);
		return diaryRepository.save(diary);
	}
```
와 같이 개선할 수 있을 것 같은데 어떻게 생각하시나요?

Reply:
조언 감사합니다! 수정했습니다

Reply:
일기 Entity에 수정한 부분이 여기엔 반영이 안됐는데 이대로 둬도 상관 없을까요?